mws/utils.py

```diff
@@ -7,38 +7,36 @@
 @author: pierre
 """
 from __future__ import absolute_import
-
+from functools import wraps
 import re
+import datetime
 import xml.etree.ElementTree as ET
 
 
-class object_dict(dict):
-    """object view of dict, you can
-    >>> a = object_dict()
+class ObjectDict(dict):
```

Comment:
I hesitated on changing the unconventional naming in case it broke API compatibility with people who are using these classes directly in their projects - although now thinking about it, maybe it can be worked around by making aliases (`object_dict = ObjectDict`)?

Reply:
I can see that being an issue for some backwards compatibility, yes. Will add that line and plan it for deprecation and removal later.